js/scripts-2.js

```diff
@@ -0,0 +1,230 @@
+let markers = [];
+let infoWindow;
+let map;
+
+/**Connecting Map Location API  */
+    /*Creating google map*/
+    function initMap() {
+        // The location of Munich
+        const munich = { lat: 48.137154, lng: 11.576124 };
+        // The map, centered at Munich
+        map = new google.maps.Map(document.getElementById("map"), {
+        //  maxZoom: 20,
+            center: munich,
+        });
+        infoWindow = new google.maps.InfoWindow()
+    }
+
+/*Connecting Supermarkets locations API from EDEKA */
+const LocationApiUrl = 'https://www.edeka.de/api/marketsearch/markets?searchstring=Munchen&size=124';
+fetch(LocationApiUrl)
+.then(response => response.json())
+.then(data => {
+    /*Calling functions */
+    displayStores(data);
+    showStoresMarker(data);
+    setOnClickListener();
+    searchByPlace();
+    // searchstores();
+});     
```

Comment:
what will happen if an error will happen in during api call or manipulation with data?